src/redux/fallbackExplorer.js

```diff
@@ -336,7 +336,7 @@ export const fetchOnchainBalances = ({
     }));
   }
 
-  if (!assets || (!assets.length && keepPolling)) {
+  if (isEmpty(assets) || (isEmpty(assets) && keepPolling)) {
```

Comment:
maybe move `isEmpty(assets)` into a bool const outside of the if

Reply:
resolving this as assets is no longer an array